test/NetworkId.purs

```diff
@@ -0,0 +1,35 @@
+module Test.Ctl.NetworkId
+  ( suite
+  ) where
+
+import Prelude
+
+import Contract.Address (addressFromBech32) as Address
+import Contract.Config (testnetConfig)
+import Contract.Monad (runContract)
+import Ctl.Internal.Plutus.Conversion.Address (fromPlutusAddress)
+import Ctl.Internal.Serialization.Address (NetworkId(..), addressBech32)
+import Effect.Aff (Aff)
+import Mote (group, test)
+import Test.Ctl.TestM (TestPlanM)
+import Test.Spec.Assertions (shouldEqual)
+
+suite :: TestPlanM (Aff Unit) Unit
+suite = group "NetworkId Tests" $ do
+  test "Testnet Address in Testnet Env" testTestnetAddress
+
+--  test "Mainnet Address in Testnet Env" testMainnetAddress
+
+testTestnetAddress :: Aff Unit
+testTestnetAddress = runContract testnetConfig do
+  let
+    bechstr =
+      "addr_test1qqm0z9quxyefwwq902p5f9t4s35smhegjthhhqpeclnpx2rzhuq2p6jahnky7qqua9nz9tcw6nlgy6cpjvmlaaye4apqzc6ppq"
+  addr <- Address.addressFromBech32 bechstr
+  addressBech32 (fromPlutusAddress TestnetId addr) `shouldEqual` bechstr
+
+-- testMainnetAddress :: Aff Unit
```

Comment:
A leftover

Reply:
are tests ok without `testMainnetAddress`? wasn't sure how to do this (see question above), but I can also just delete it